### PR TITLE
c_ares, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
@korli could you review, first time I'm doing it this way by keeping an older recipe alive :)

**Comment by korli** [comment]
> @korli could you review, first time I'm doing it this way by keeping an older recipe alive :)

If the SONAME is the same, both versions can't be installed side by side.

**Comment by Begasus** [comment]
Indeed the same, should remember to check this when something similar passes by again :)

File: net-dns/c_ares/c_ares17-1.17.2.recipe [hunk 1]
```diff
@@ -0,0 +1,67 @@
+SUMMARY="A C library for asynchronous DNS requests"
```
[the remark this answers is not in the document]
Comment (Begasus) [reply net-dns/c_ares/c_ares17-1.17.2.recipe:1]: Will be removed, thanks for the pointer though!

**Pull request merged**